tests/foreman/virtwho/cli/test_esx.py

```diff
@@ -471,3 +471,46 @@ def test_positive_deploy_configure_hypervisor_password_with_special_characters(
         )
         target_sat.cli.VirtWhoConfig.delete({'name': virtwho_config['name']})
         assert not target_sat.cli.VirtWhoConfig.exists(search=('name', form_data['name']))
+
+    @pytest.mark.tier2
+    def test_positive_remove_env_option(self, default_org, form_data, virtwho_config, target_sat):
+        """remove option 'env=' from the virt-who configuration file and without any error
+
+        :id: 509add77-dce7-4ba4-b9e5-2a5818c39731
+
+        :expectedresults:
+            the option "env=" should be removed from etc/virt-who.d/virt-who.conf
+            /var/log/messages should not display warning message
+
+        :CaseLevel: Integration
+
+        :CaseImportance: Medium
+
+        :BZ: 1834897
+
+        :customerscenario: true
+        """
+        command = get_configure_command(virtwho_config['id'], default_org.name)
+        deploy_configure_by_command(
+            command, form_data['hypervisor-type'], debug=True, org=default_org.label
+        )
+        virt_who_instance = target_sat.cli.VirtWhoConfig.info({'id': virtwho_config['id']})[
+            'general-information'
+        ]['status']
+        assert virt_who_instance == 'OK'
```

Comment:
same here, Expect this is "Caps" ?

Reply:
> same here, Expect this is "Caps" ?

@ColeHiggins2 The status of virt-who config get from API and CLI are different, it is lowercase in api, and Caps in CLI.

From CLI ```{'general-information': {'id': '107', 'name': 'KCqBdpanvN', 'hypervisor-type': 'esx', 'hypervisor-server': 'XX.xX.XX.XX'', 'hypervisor-username': 'XXX', 'status': 'OK'}, 'schedule': {'interval': 'every hour', 'last-report-at': '2022/10/09 01:22:18'}, 'connection': {'satellite-server': 'dell-per740-68-vm-09.lab.eng.pek2.redhat.com', 'hypervisor-id': 'hostname', 'filtering': 'Unlimited', 'debug-mode': 'yes'}, 'http-proxy': {}}
```
From API:
```[robottelo.hosts.DecClass(blacklist=None, debug=True, exclude_host_parents=None, filter_host_parents=None, filtering_mode=0, http_proxy=None, hypervisor_id='hostname', hypervisor_server='XX.xX.XX.XX', hypervisor_type='esx', hypervisor_username='XXX', interval=60, name='WAKRsXTTMT', no_proxy=None, organization_id=1, satellite_url='dell-per740-68-vm-09.lab.eng.pek2.redhat.com', status='ok', whitelist=None, prism_flavor=None, kubeconfig_path=None, id=119)]

```